### ISSUE TITLE
[2.1] Volt compilation bug

### ISSUE THREAD

**Issue by bluetec** [phalcon/cphalcon#10876]
Just after upgrading to phalcon 2.1 beta, i got errors in volt compiler

```
Notice: Array to string conversion in C:\xampp\htdocs\PhaconSkeleton\app\bootstrap\Kernel.php on line 23
 Ops! Template file Arraylayout/pages.volt does not exist#0 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compileFile('Arraylayout/pag...', 'C:\\xampp\\htdocs...', true)
 #1 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compile('Arraylayout/pag...', true)
 #2 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->_statementList(Array, false)
 #3 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->_compileSource('{% extends "lay...', false)
 #4 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compileFile('C:\\xampp\\htdocs...', 'C:\\xampp\\htdocs...', false)
 #5 [internal function]: Phalcon\Mvc\View\Engine\Volt\Compiler->compile('C:\\xampp\\htdocs...')
 #6 [internal function]: Phalcon\Mvc\View\Engine\Volt->render('C:\\xampp\\htdocs...', Array, true)
 #7 [internal function]: Phalcon\Mvc\View->_engineRender(Array, 'user/login', true, true, NULL)
 #8 [internal function]: Phalcon\Mvc\View->render('user', 'login', Array)
 #9 C:\xampp\htdocs\PhaconSkeleton\app\bootstrap\Kernel.php(23): Phalcon\Mvc\Application->handle()
 #10 C:\xampp\htdocs\PhaconSkeleton\app\bootstrap\Base.php(109): App\Bootstrap\Kernel->afterRun(Array)
 #11 C:\xampp\htdocs\PhaconSkeleton\public\index.php(46): App\Bootstrap\Base->run()
 #12 {main}
```

After review Phalcon source code, i just found this bugs.

File: phalcon/mvc/view/engine/volt/compiler.zep
Line: 1820 & 2096
```
if typeof view == "object" {
				
	// Bug: view->getViewsDir returns array
	// Array to string conversion !
	let finalPath = view->getViewsDir() . path;
} else {
	let finalPath = path;
}
```

**Comment by KorsaR-ZN** [comment]
Today I send a patch fixes this and some other bugs 2.1.x branch

**Comment by andresgutierrez** [comment]
Fixed in 2.1.x branch
